lib/testmap.py

```diff
@@ -189,16 +181,13 @@
     },
     'candlepin/subscription-manager-cockpit': {
         'main': [
-            'rhel-9-1',
             'centos-9-stream',
             'rhel-9-2',
             'fedora-36',
             'fedora-37',
         ],
         '_manual': [
             'centos-8-stream/subscription-manager-1.28',
-            'rhel-8-7',
-            'rhel-8-7/subscription-manager-1.28',
```

Comment:
can you please move the testing using the sub-man-1.28 branch to 8.9?

Reply:
You mean for regularly testing it, instead of only `_manual`? As it already is on `_manual`, and so is this removal. But sure, let me check whether this works -- I triggered "rhel-8-9@candlepin/subscription-manager/subscription-manager-1.28" here. This PR needs work anyway, due to [this](https://cockpit-logs.us-east-1.linodeobjects.com/pull-4659-20230417-045930-0a83a092-rhel4edge-cockpit-project-cockpit/log.html).

Reply:
`_manual`-only is fine, yes

Reply:
[works](https://cockpit-logs.us-east-1.linodeobjects.com/pull-4659-20230417-050759-0a83a092-rhel-8-9-candlepin-subscription-manager-subscription-manager-1.28/log.html), added!

Reply:
It already was on manual. But I suppose it doesn't hurt to regularly test it on 8-9?

Reply:
Sorry, I was not clear: what I meant was replacing `rhel-8-7/subscription-manager-1.28` (ie with the old sub-man branch) with `rhel-8-9/subscription-manager-1.28`, still in `_manual`; there is already `rhel-8-9` (ie with sub-man/main) in `_manual`

Reply:
... thinking about it further, considering there is already `centos-8-stream/subscription-manager-1.28` there, it should be OK for now (and new combinations can always be tested locally anyway)

tl;dr: changes are fine, thanks!

Reply:
Now I understand -- you mean "rhel-8-7/subscription-manager-1.28" is a scenario actually understood by sub-man. Yes, I'll do that then. Updated!